NOTICE-binary

```diff
@@ -1236,7 +1236,7 @@ This product optionally depends on 'zstd-jni', a zstd-jni Java compression
 and decompression library, which can be obtained at:
 
   * LICENSE:
-    * license/LICENSE.zstd-jni.txt (Apache License 2.0)
```

Comment:
Thanks for fixing this license issue.

Reply:
🍻 No problem. And we could make this change to a separate PR if you want.